src/Features/Core/Portable/EmbeddedLanguages/RegularExpressions/RegexEmbeddedCompletionProvider.cs

```diff
@@ -47,21 +49,7 @@ public override bool ShouldTriggerCompletion(SourceText text, int caretPosition,
 
             if (trigger.Kind == CompletionTriggerKind.Insertion)
             {
-                return IsTriggerCharacter(trigger.Character);
-            }
-
-            return false;
-        }
-
-        private static bool IsTriggerCharacter(char ch)
-        {
-            switch (ch)
-            {
-                case '\\': // any escape
-                case '[':  // character class
-                case '(':  // any group
-                case '{':  // \p{
```

Comment:
woudl have liked these comments to remain in `=> ImmutableHashSet.Create('\\', '[', '(', '{');`